src/compiler/commandLineParser.ts

```diff
@@ -281,9 +281,14 @@ namespace ts {
             description: Diagnostics.Disallow_inconsistently_cased_references_to_the_same_file
         },
         {
+            name: "allowSyntheticDefaultImports",
+            type: "boolean",
+            description: Diagnostics.Allow_default_imports_from_modules_with_no_default_export

```

Comment:
do we need to add more description that we are not adding anything to the emitted code, but that it is just an illusion to match what the loader would do.